### DISCUSSION TITLE
Duplicate record [Oct 06]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#4069] [Discord Threads]
> 🙋‍♂️ **HugoBarjot**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### Not possible at the moment.  
#  
> 🙋‍♂️ **HugoBarjot**  
#### Ok  thx  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1025025497052229712)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
